ci/common_install.sh

```diff
@@ -122,7 +122,19 @@ fi
 
 # Setup ccache
 if [ "$B2_USE_CCACHE" == "1" ]; then
-    "$CI_DIR"/setup_ccache.sh
+    if ! "$CI_DIR"/setup_ccache.sh 2>&1; then
+        set +x
```

Comment:
Is echo necessary here?

Reply:
I wanted to make it as visible as possible so enclosing this message between an empty line sounded like a good idea, see https://github.com/boostorg/boost-ci/actions/runs/12505996558/job/34890259844?pr=254#step:10:310

Doesn't hurt, does it?